test/libponyc-run/runner.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+pushd () {
+    command pushd "$@" > /dev/null
+}
+
+popd () {
+    command popd "$@" > /dev/null
+}
+
+exit_code=0
+
+for directory in $(find ./ -maxdepth 1 -type d ! -iname ".*")
+do
+  pushd ${directory}
```

Comment:
pushd and popd are wonderful but aren't Posix. I can adjust this to be either Posix or bash as we decide.

Reply:
it was decided to go for bash.